controllers/topics.controller.js

```diff
@@ -0,0 +1,17 @@
+const { selectAllTopics } = require("../models/topics.model")
+
+
+function getAllTopics(req, res){
+    console.log(selectAllTopics)
+    return selectAllTopics()
+    .then((topics) =>{
+        res.status(200).send({topics})
+    })
+    .catch((err) =>{
+        res.status(500).send({msg:"Error whilst fetching parks data"})
```

Comment:
your error handling should be extracted into error handling middleware functions.

Reply:
In this case this would be a handleServerError error handling middleware function that would return a 500 status code and a message along the lines of "server error" - in this error handler you could also console.log the current error to help you debug it further.